mitmproxy/addons/core.py

```diff
@@ -50,18 +50,19 @@ def configure(self, updated):
                     )
 
     @command.command("set")
-    def set(self, option: str, value: str = "") -> None:
+    def set(self, option: str, *value: str) -> None:
         """
             Set an option. When the value is omitted, booleans are set to true,
             strings and integers are set to None (if permitted), and sequences
             are emptied. Boolean values can be true, false or toggle.
             Multiple values are concatenated with a single space.
-            Sequences are set using multiple invocations to set for
-            the same option.
```

Comment:
This explanation here was bonkers (and never worked), the previous line already explains how to set multiple values.